design_docs/Annotation.md

```diff
@@ -0,0 +1,54 @@
+# Annotation
+
+Prior to running the filtering and categorisation process, we need the variant data to be annotated; population
+frequencies, clinvar entries, and a range of predicted consequences. To do this we use a combination of VEP and
+annotation directly from Hail Tables containing annotations.
+
+This process is split into a number of steps:
+
+1. If the variant (singleton, or joint call) data is provided in MatrixTable format, we revert this back to a VCF
+2. We use GATK to identify intervals across the genome, so the VCF can be evenly split and annotation tasks parallelised
```

Comment:
"evenly". Ideally we want dynamic batches, so we can split into regions based on amount data, and not based on pre-defined fixed intervals, so it would be much more even :) But that's down the road